spec/std/float_printer_spec.cr

```diff
@@ -106,7 +106,7 @@ describe "#print Float64" do
     test_pair 5.5626846462680035e-309, "5.562684646268003e-309"
   end
 
-  it "falure case" do
+  pending_win32 "failure case" do
```

Comment:
Why is this broken?

Reply:
I have no idea. It returns a different result. Maybe that's expected? IDK. Needs investigation into the win32 libs.

Reply:
It seems there is something fundamentally broken with floating point numbers on win32:
```cr
"%f" % 1.0 # => "0.000000"
```